lib/i18n/payment-method-type.js

```diff
@@ -39,6 +39,8 @@ const i18nTypes = defineMessages({
   SUBSCRIPTION: {
     defaultMessage: 'PayPal subscription',
   },
+  US_BANK_ACCOUNT: { id: 'ACH Debit', defaultMessage: 'ACH Debit' },
+  SEPA_DEBIT: { id: 'SEPADebit', defaultMessage: 'SEPA Debit' },
```

Comment:
Fixes this:
![image](https://user-images.githubusercontent.com/2119706/203875391-aee6ab75-772d-41b1-8ff9-9db8695c557b.png)